app/src/main/java/com/example/practicals/activity_intent_fragment_practicals/Activities/JetpackNavActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.practicals.activity_intent_fragment_practicals.Activities
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.navigation.findNavController
+import androidx.navigation.ui.setupWithNavController
+import com.example.practicals.R
+import kotlinx.android.synthetic.main.activity_jetpack_nav.*
+
+class JetpackNavActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_jetpack_nav)
+
+        bottomNavBind()
```

Comment:
use proper name for functions rather then shortnames, like bindBottomNavigation, or setupBottomNavigation